readme.md

```diff
@@ -0,0 +1,109 @@
+# GitPress Example
+
+The objective here is to create example project for the ![Gitpress platform](https://github.com/pschoffer/gitpress).
+Its very easy to install and you can try it easily home.
+
+## Instalation
+
+ ### Manual Instalation  	
```

Comment:
Please drop the manual instalation. Have you tried if that approach actually works?

The thing is that GitPress has "integrated" database. It is not meant to connect to any other database, these steps are addressing configuring wordpress with database and creating DB schema which is something this project is aiming to avoid. (by doing it in kinda integrated transparent way)

Reply:
I got it. 
Its a very nice project, simplifies a lot the Wordpress experience. It reminds me of Fission codes platform on IPFS that let you build you front page without bothering about the backend and the publishing.

Reply:
Great:) Yeah to be honest I don't know much about WordPress and I knew much less when I started this. But on the other hand, this is just mashing together that awesome VersionPress plugin, WordPress and some docker goodies to standalone thing:)

You know, the regular "standing on the shoulders of giants" approach. It is awesome that it got you interested in contributing \o/.